dashboards-observability/common/constants/autocomplete.ts

```diff
@@ -1,8 +1,11 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Can this be moved below the copyright notice?

Reply:
Most definitely can

Reply:
There are a bunch of other files that have it above the copyright notice, should I change those as well?

Reply:
Yes, the copyright header should be at the top of the file 👍

Reply:
yes, copyright should be at the top